templates_jinja2/team_partials/team_info.html

```diff
@@ -15,6 +15,7 @@
       {% if team.location %}<span class="glyphicon glyphicon-map-marker"></span> From <a href="http://maps.google.com/maps?q={{ team.location|urlencode }}" target="_blank">{{ team.location }}</a><br>{% endif %}
       {% if team.name %}<span class="glyphicon glyphicon-info-sign"></span> aka <em>{{ team.name }}</em><br>{% endif %}
       {% if team.rookie_year %}<span class="glyphicon glyphicon-calendar"></span> First competed in {{ team.rookie_year }}<br>{% endif %}
+      {% if last_competed and year != current_year %}Last competed in {{ last_competed }}.{% endif %}
```

Comment:
Fixed in https://github.com/the-blue-alliance/the-blue-alliance/commit/5524b71c589a2ac2ab95b5ff75c2d3385fec69b6